calour/transforming.py

```diff
@@ -370,3 +370,17 @@ def subsample_count(exp: Experiment, total, replace=False, inplace=False, random
     exp.reorder([i not in drops for i in range(exp.data.shape[0])], inplace=True)
     exp.normalized = total
     return exp
+
+
+def _subsample(data, depth):
```

Comment:
do you want this in this PR or it is just an accidental commit? if so, could you add a test?

Reply:
oops wrong branch... deleted